portugues_pt/colaborar.md

```diff
@@ -16,6 +16,8 @@ Compraria [componentes eletronicos](componentes/componentes.md)
 
 Revolucionava o sistema de [ensino](ensino/alternativo.md)
 
+Mandaria ladrilhar a rua toda pra [deixar a cidade mais bonita](ladrilho/ladrilhar.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.